pkgs/by-name/pl/plutovg/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+stdenv.mkDerivation rec {
+  pname = "plutovg";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "sammycage";
+    repo = "plutovg";
+    tag = "v${version}";
```

nit: generally finalAttrs is preferred to rec
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "plutovg";
  version = "1.0.0";

  src = fetchFromGitHub {
    owner = "sammycage";
    repo = "plutovg";
    tag = "v${finalAttrs.version}";
```